packages/grpc-js/src/call-stream.ts

```diff
@@ -112,6 +112,15 @@ export class Http2CallStream extends Duplex implements Call {
   // This is populated (non-null) if and only if the call has ended
   private finalStatus: StatusObject | null = null;
 
+  private socketCloseCallback: () => void = 
```

Comment:
Should this be something like the following?

```js
private socketCloseCallback = () => {
  this.endCall(...);
};
```

Reply:
Yes, that's a copy/paste error.